dist/rpm/openQA.spec

```diff
@@ -646,8 +648,8 @@ fi
 %dir %{_datadir}/openqa/script
 %{_datadir}/openqa/script/client
 %{_datadir}/openqa/script/clone_job.pl
-%{_datadir}/openqa/script/dump_templates
-%{_datadir}/openqa/script/load_templates
```

Comment:
The removal looks wrong considering

```
ln -s openqa-dump-templates %{_datadir}/openqa/script/dump_templates
ln -s openqa-load-templates %{_datadir}/openqa/script/load_templates
```

still creates those symlinks. However, you could likely remove *those* symlinks as well (unless we want to keep them for compatibility).

Reply:
since @okurz said that backwards compatibility is desired below, I've set this as resolved